client/src/components/Image.jsx

```diff
@@ -20,58 +21,68 @@ class Image extends React.Component {
     if (this.props.showAd) {
       const { showModal } = this.state;
       return (
-        <React.Fragment>
-          <div
-            className="modal-toggle-button"
-            onClick={this.toggleModal}
-          >
+        <>
+          <FeedbackWrapper className="modal-toggle-button" onClick={this.toggleModal}>
             Feedback
-          </div>
+          </FeedbackWrapper>
           {
             showModal ? (
-              <Modal>
-                <h1>Share your feedback</h1>
-                <img src={this.props.photo.imageUrl}></img>
-                <div>{this.props.photo.title}</div>
-                <div>${this.props.photo.price}</div>
-                <form action="">
-                  <label>This item is:</label><br />
-                  <input type="radio" name="feedback" value="male" /> Unrelated to what I'm shopping for<br />
-                  <input type="radio" name="feedback" value="female" /> Inappropriate or offensive <br />
-                  <input type="radio" name="feedback" value="other" /> Other
-                </form>
-                <form action="/action_page.php">
-                  <label>Comments</label> <br />
-                  <textarea rows="4" cols="50">
-                  </textarea>
-                  <br />
-                  <button>Cancel</button>
-                  <input type="submit" value="Send feedback" />
-                </form>
-                <button className="modal-close" onClick={this.toggleModal}>X</button>
-              </Modal>
+                <Modal>
+                  <ModalOverlay>
+                  <ModalWrapper>
+                    <ModalHeader>Share your feedback</ModalHeader>
+                    <PhotoContainer src={this.props.photo.imageUrl}></PhotoContainer>
+                    <FloatTitlePrice>{this.props.photo.title}
+                      <br />
+                      <PriceColor>${this.props.photo.price}</PriceColor>
+                    </FloatTitlePrice>
+                    <br />
+                    <FormsContainer>
```

Comment:
Styling Looks dope! You could also just call it FeedbackForm or something if you wanted to be more semantic than container but yeah this all looks greeeaaat